test/e2e/bootstrap/OWNERS

```diff
@@ -0,0 +1,13 @@
+approvers:
+- jbeda
+- luxas
+- wanghaoran1988
+reviewers:
+- mikedanese
+- luxas
+- justinsb
```

Comment:
you can remove justinsb, errordeveloper and lukemarsden here as they weren't involved in this feature and likely don't want to receive notifications for this